integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java

```diff
@@ -62,14 +62,12 @@
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.concurrent.TimeUnit.SECONDS;
 import static oracle.weblogic.kubernetes.TestConstants.BASE_IMAGES_REPO_SECRET;
-import static oracle.weblogic.kubernetes.TestConstants.DB_OPERATOR_YAML_URL;
+import static oracle.weblogic.kubernetes.TestConstants.DB_IMAGE_NAME;
+import static oracle.weblogic.kubernetes.TestConstants.DB_OPERATOR_IMAGE;
+import static oracle.weblogic.kubernetes.TestConstants.OCIR_SECRET_NAME;
 import static oracle.weblogic.kubernetes.TestConstants.OCR_DB_19C_IMAGE_TAG;
```

Comment:
Rename it to OCIR_DB_19C_IMAGE_TAG  or simple DB_19C_IMAGE_TAG

Reply:
We have other images with OCR prefix.

  public static final String OCR_WEBLOGIC_IMAGE_NAME = "middleware/weblogic";
  public static final String OCR_WEBLOGIC_IMAGE_TAG = "12.2.1.4";
  public static final String OCR_FMWINFRA_IMAGE_NAME = "middleware/fmw-infrastructure";
  public static final String OCR_FMWINFRA_IMAGE_TAG = "12.2.1.4";
  public static final String OCR_DB_IMAGE_NAME = "database/enterprise";
  public static final String OCR_DB_IMAGE_TAG = "12.2.0.1-slim";
  public static final String OCR_DB_19C_IMAGE_TAG = "19.3.0.0";

Reply:
We will remove them when we will consolidate the environment variables.